versions/3.0.md

```diff
@@ -921,35 +921,29 @@ A request body with a referenced model definition.
 ```json
 {
   "description": "user to add to the system",
-  "content" : {
-    "application/json" : {
+  "content": {
+    "application/json": {
       "schema": {
-          "$ref": "#/definitions/User"
-        },
-      "examples": [
-        { "$ref": 'http://foo.bar#/examples/address-example.json'}
-      ]
+        "$ref": "#/definitions/User"
+      },
+      "examples": [ "http://foo.bar/examples/user-example.json" ]
     },
-    "application/xml" : {
-       "schema": {
+    "application/xml": {
+      "schema": {
         "$ref": "#/definitions/User"
       },
-      "examples": [
-        { "$ref": 'http://foo.bar#/examples/address-example.xml'}
-      ]
+      "examples": [ "http://foo.bar/examples/user-example.xml" ]
     },
-    "text/plain" : {
-      "examples": [
-        { "$ref": 'http://foo.bar#/examples/address-example.txt'}
-      ]
+    "text/plain": {
+      "examples": [ "http://foo.bar/examples/user-example.txt" ]
     },
     "*": {
-      "example": {
-        $ref: "http://foo.bar#/examples/address-example.whatever"
+      "schema": {
```

Comment:
I replaced this JSON completely, generating it from the YAML (because the two did not match; one used foo.bar# and the other foo.bar. I'll also correct the YAML
```

  '*':
    schema:
      $ref: 'http://foo.bar/examples/user-example.whatever'
```
to
```

  '*':
    example:
      $ref: 'http://foo.bar/examples/user-example.whatever'
```
with the update